contracts/VToken.sol

```diff
@@ -463,8 +567,58 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
         totalBorrows = totalBorrowsNew;
         totalReserves = totalReservesNew;
 
+        uint256 err = _accrueStableInterest(cashPrior, reservesPrior, blockDelta);
+
+        if (err != 0) {
+            return err;
+        }
+
         /* We emit an AccrueInterest event */
-        emit AccrueInterest(cashPrior, interestAccumulated, borrowIndexNew, totalBorrowsNew);
+        emit AccrueInterest(cashPrior, interestAccumulated, borrowIndexNew, totalBorrowsNew, stableBorrowIndex);
+
+        return NO_ERROR;
+    }
+
+    /**
+     * @notice Applies accrued stable interest to stable borrows and reserves
+     * @dev This calculates interest accrued from the last checkpointed block
+     *   up to the current block and writes new checkpoint to storage.
+     * @param cashPrior total available cash
```

Comment:
```suggestion
     * @param cashPrior Total available cash
```

Reply:
Fixed, can check the changes at [this commit](2981e0d08b247b7c7a7dc3461e9701d7d593c520).